### ISSUE TITLE
about the zLevel between text and mask for echartsInstance.showLoading()

### ISSUE THREAD

**Issue by vvizden** [apache/echarts#14386]
### Version
5.0.2

### Steps to reproduce
```js
      chartInstance.showLoading({
        text: 'Nothing',
        color: '#ffffff',
        textColor: '#909399',
        maskColor: 'rgba(255, 255, 255, 1)',
        showSpinner: false,
      })
```

### What is expected?
The text is above the mask.

### What is actually happening?
The mask covers the text.

---
For the version ^4.8.0, it is expected.

**Comment by plainheart** [comment]
Thanks for the feedback! We've fixed it in #14191. Please be patient to wait for the next release.

**Comment by vvizden** [comment]
> Thanks for the feedback! We've fixed it in #14191. Please be patient to wait for the next release.

Sorry, I didn't find the similar issue at the first time. I'll close this issue.
